scipy/linalg/_decomp.py

```diff
@@ -976,7 +977,7 @@ def eigvalsh(a, b=None, lower=True, overwrite_a=False,
     Returns
     -------
     w : (N,) ndarray
-        The ``N`` (``1<=N<=M``) selected eigenvalues, in ascending order, each
+        The N (N<=M) selected eigenvalues, in ascending order, each
```

Comment:
You need those double backticks to mark things as inline code

Reply:
I think I was just following `eigh` here, which doesn't use backticks. In general it seemed a bit random to me where double backticks are used and where they're not.